### PR TITLE
Added New Roles and Updated Roles/Modifiers

### PR DESCRIPTION
**Pull request by R9H9**
I added Neighbor, Nurse and Gramps.
Edit: I also added Commuter, Lookout and Scout.

### CODE REVIEW

File: Games/types/Mafia/const/Priority.js [hunk 1]
```diff
@@ -39,9 +41,13 @@ const PRIORITY_GUN_GIVER = PRIORITY_ITEM_GIVER_DEFAULT
 const PRIORITY_ORANGE_GIVER = PRIORITY_ITEM_GIVER_DEFAULT
 const PRIORITY_PROBE = PRIORITY_ITEM_GIVER_DEFAULT
 
+const PRIORITY_MESSAGE_GIVER_DEFAULT = -50
+const PRIORITY_CONFIRM_SELF = PRIORITY_MESSAGE_GIVER_DEFAULT
+
 const PRIORITY_INVESTIGATIVE_DEFAULT = -10
 const PRIORITY_ALIGNMENT_LEARNER = PRIORITY_INVESTIGATIVE_DEFAULT
 const PRIORITY_ROLE_LEARNER = PRIORITY_INVESTIGATIVE_DEFAULT
+const PRIORITY_LEARN_VISITORS = PRIORITY_INVESTIGATIVE_DEFAULT
```
Comment (pikulet) [inline Games/types/Mafia/const/Priority.js:50]: I think learn visitors has to be the last action after tracker and watcher, or gramps cannot see tracker and watcher move.

**Comment by pikulet** [comment]
hi, should we have distinct priorities for role blockers? 

escort, hooker, nurse

but i don't know what the order should be

having distinct priorities is not important for sandbox but I think it's helpful for comp/main

**Comment by r3ndd** [comment]
Wow, this is a lot of contributions! Thank you for this.

**Comment by r3ndd** [comment]
> Are the roles supposed to be tested before or after a merge?

Ideally before. I'm currently fixing the issues though.

**Comment by r3ndd** [comment]
Okay I think it's ready for prod

**Pull request merged**